### PR TITLE
Add basic operations page

### LINKED ISSUE #110: install-manual page needs some steps added
After the `juju deploy` commands, the user will be left wondering what to do next. We don't have instructions here for:

1. `watch -c juju status --color`
2. installing the kubectl snap
3. downloading the cluster kubeconfig with `juju scp`
4. accessing the dashboard

We should bring over the instructions we have for these steps at https://jujucharms.com/canonical-kubernetes/ and include them in the install-manual page after the "Deploy the CDK bundle" section.

### PR DESCRIPTION
**Pull request by evilnick**
@tvansteenburgh This basically fixes #110 IMO, and draws together the manual, quickstart and potentially any other install pages we add (they can all point here).

There are parts which can be improved for sure, but I think it would be good to get this in with the 1.14 stuff (and helps with the "try this version" story).

### CODE REVIEW

File: pages/k8s/operations.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+some basic cluster operations, including how to check the status of your cluster and test
+that it is working with the built-in demo deployment.
+
+If you are already familiar with this, you may still like to check out the [Next steps][next]
+for more useful guides on operating CDK.
+
+## Install and configure kubectl
+
+You will need **kubectl** to be able to use your Kubernetes cluster. If it is not already
+installed (it is automatically  intalled if you used conjure-up to deploy **CDK**), it is easy
```
Comment (tvansteenburgh) [inline pages/k8s/operations.md]: installed

File: pages/k8s/operations.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+  description: How to operate your CDK cluster.
+keywords: dashboard, juju,kubectl,config
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: operations.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+Now you have installed your **CDK** cluster, what can you do with it? This page details
```
Comment (tvansteenburgh) [inline pages/k8s/operations.md]: Now that you have (right?)

File: pages/k8s/operations.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+[nginx-ingress-controller](https://github.com/kubernetes/contrib/tree/master/ingress/controllers/nginx)
+project on github.
+
+
+
+<a id='next'> </a>
+
+## Next steps
+
+Now your **CDK** cluster is up and running, here are some suggestions for additional
```
Comment (tvansteenburgh) [inline pages/k8s/operations.md]: Again, should this be "Now that your..." instead? Maybe this is a British/American English thing, I dunno.

Comment (evilnick) [reply pages/k8s/operations.md]: either would be okay in GB (i.e. real) English, but if it sounds odd to your ears, I will add a few 'that's

File: pages/k8s/operations.md (hunk 1 above)
Comment (evilnick) [reply pages/k8s/operations.md]: This was obviously the one typo I habitually leave to make you feel you have accomplished something :smiley:

**Pull request merged**